### PR TITLE
 CSS changes acc to Issue#948

### LINKED ISSUE #948: [OTHER] Use prime react classes for button container width
### What would you like to share?

1. Remove the following CSS from Links.css: 
Screenshot 2022-01-09 161100

2. Replace with sm:w-9 md:w-7 on line 23 of Links.js 
Screenshot 2022-01-09 161125

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by vish-han**
## Fixes Issue
#948 

Closes #948 
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [X] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots

Removed This piece of code from Links.css 
https://user-images.githubusercontent.com/57045550/148688461-efae8a09-e7df-41f0-b003-e6fbef402e23.png
Added this code in Links.js
https://user-images.githubusercontent.com/57045550/148688485-54d1f9ae-5859-4b4a-a921-e6ee23871cae.png

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1189"></a>

### CODE REVIEW

**Review by emmalearnscode — CHANGES_REQUESTED** [review]
Hi Vishal! I just merged your profile in the other PR you created but unfortunately it has created a conflict in this PR, otherwise the CSS file looks good. If you can solve the merge conflict and let us know when you're done we can take another look at this PR and get it merged. If you need help with the conflict don't hesitate to ask :)

File: public/data/vishalchauhan.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "Vishal Chauhan",
   "type": "personal",
-  "bio": "OpenSource Enthusiast interested in Cloud Native Technologies",
+  "bio": "OpenSource Enthusiast interested in Cloud Native Technologies
```
Comment (schmelto) [inline public/data/vishalchauhan.json:4]:
```suggestion
  "bio": "OpenSource Enthusiast interested in Cloud Native Technologies",
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by schmelto — CHANGES_REQUESTED** [review]

**Comment by vish-han** [comment]
@schmelto  and @emmalearnscode  i have tried to resolve the merge conflict , can you please review the PR now.😊

**Review by schmelto — APPROVED** [review]
LGTM.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Efficient

**Review by krshdev — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
LGTM, thank you for your contribution :rocket: :smile: 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Efficient, Elegant

**Review by emmalearnscode — APPROVED** [review]

**Pull request merged**